### PR TITLE
feat: captcha

### PR DESCRIPTION
**Pull request by iGroza**
Fixes #

## Proposed Changes

  - implement captcha modal;

## Checklist

- [x] I have read the rules of [**contribution**](https://github.com/haqq-network/haqq-wallet/blob/main/docs/contribution.md#important-steps).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,7 +24,9 @@
     "@haqq/provider-mnemonic-react-native": "0.0.5",
     "@haqq/provider-mpc-react-native": "0.0.2",
     "@haqq/provider-web3-utils": "0.0.9",
+    "@hcaptcha/react-native-hcaptcha": "^1.5.1",
     "@invertase/react-native-apple-authentication": "^2.2.2",
+    "@hcaptcha/react-native-hcaptcha": "^1.5.1",
```
Comment (vivalaakam) [inline package.json:29]: duplicate

File: package.json [hunk 2]
```diff
@@ -78,18 +80,21 @@
...
     "react-native-linear-gradient": "2.6.2",
     "react-native-markdown-display": "7.0.0-alpha.2",
+    "react-native-modal": "^13.0.1",
     "react-native-os": "https://github.com/aprock/react-native-os#938676ebd92d58b8928168a7e78cbe216838ed17",
     "react-native-prompt-android": "1.1.0",
     "react-native-qr-decode-image-camera": "1.1.2",
     "react-native-qrcode-svg": "6.2.0",
     "react-native-randombytes": "3.6.1",
     "react-native-reanimated": "2.14.4",
+    "react-native-recaptcha-that-works": "^1.3.2",
```
Comment (vivalaakam) [inline package.json:90]: why there 2 captchas ?

File: package.json [hunk 3]
```diff
@@ -78,18 +80,21 @@
     "react-native-keychain": "8.1.1",
     "react-native-linear-gradient": "2.6.2",
     "react-native-markdown-display": "7.0.0-alpha.2",
+    "react-native-modal": "^13.0.1",
```
[the remark this answers is not in the document]
Comment (iGroza) [reply package.json:83]: merge conflicts error

File: src/env.d.ts [hunk 4]
```diff
@@ -15,6 +15,8 @@ declare module '@env' {
   export const WEB3AUTH_AUTH0_CLIENT_ID: string;
   export const WEB3AUTH_AUTH0_VERIFIER: string;
   export const WEB3AUTH_AUTH0_DOMAIN: string;
+  export const HCAPTCHA_SITE_KEY: string;
   export const WEB3AUTH_DISCORD_VERIFIER: string;
   export const WEB3AUTH_DISCORD_CLIENT_ID: string;
+  export const HCAPTCHA_SITE_KEY: string;
```
Comment (vivalaakam) [inline src/env.d.ts:21]: duplicate

**Review by vivalaakam — APPROVED** [review]

**Pull request merged**